packages/common/src/core/schemas/internal/discussions/EntityUiSchemaDiscussionsSettings.ts

```diff
@@ -0,0 +1,148 @@
+import type { IArcGISContext } from "../../../../types/IArcGISContext";
+import { IUiSchema } from "../../types";
+import { EntityEditorOptions } from "../EditorOptions";
+
+/**
+ * Builds the UI Schema for the "settings => discussions" workspace pane
+ * @param i18nScope the i18n scope for translations
+ * @param options an EntityEditorOptions object
+ * @param context an IArcGISContext object
+ * @returns a promise that resolves a UI Schema object
+ */
+export const buildUiSchema = async (
+  i18nScope: string,
+  options: EntityEditorOptions,
+  context: IArcGISContext
+): Promise<IUiSchema> => {
+  const uiSchema: IUiSchema = {
+    type: "Layout",
+    elements: [
+      {
+        type: "Section",
+        labelKey: "shared.sections.discussionSettings.moderation.label",
+        elements: [
+          {
+            labelKey: "shared.fields.discussable.label",
+            scope: "/properties/isDiscussable",
+            type: "Control",
+            options: {
+              control: "hub-field-input-tile-select",
+              labels: [
+                "{{shared.fields.discussable.enabled.label:translate}}",
+                `{{${i18nScope}.fields.discussable.disabled.label:translate}}`,
+              ],
+              descriptions: [
+                "{{shared.fields.discussable.disabled.label:translate}}",
+                `{{${i18nScope}.fields.discussable.disabled.description:translate}}`,
+              ],
+              icons: ["speech-bubbles", "circle-disallowed"],
+              layout: "horizontal",
+              styles: { "max-width": "45rem" },
+              type: "radio",
+            },
+          },
+        ],
+      },
+    ],
+  };
+  // if (context.hubLicense === "hub-premium") {
```

Comment:
This is intentionally commented out for now. We don't want to expose the `allowed channel ids` field in Discussion Board workspace `Settings => Discussions` pane quite yet. That will occur when we're ready to completely cut over from Discussions API V1 to Discussions API V2. [PR-1853](https://github.com/Esri/hub.js/pull/1853) is a longer-living branch that will include the work to complete that cutover; that branch uncomments this. (That branch also includes _all_ of this work, the diff on that branch will get significantly smaller once this PR merges to master).